src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheckTest.java

```diff
@@ -297,6 +297,14 @@ public void testUnusedLocalVariableTernaryAndExpressions() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testUnusedLocalVariable2() throws Exception {
```

Comment:
Could you rename it to `testUnusedLocalVariableSwitchStatement`?

Reply:
Done